docs/management/connectors/action-types/webhook.asciidoc

```diff
@@ -23,49 +23,25 @@ image::management/connectors/images/webhook-connector.png[Webhook connector]
 Webhook connectors have the following configuration properties:
 
 Name::      The name of the connector.
-URL::       The request URL. If you are using the <<action-settings, `xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
-Method::    HTTP request method, either `post`(default) or `put`.
-Headers::   A set of key-value pairs sent as headers with the request
-Require authentication:: If true, a username and password for login type authentication must be provided.
-Username::      Username for HTTP basic authentication.
-Password::  Password for HTTP basic authentication.
-
-[float]
-[[preconfigured-webhook-configuration]]
-=== Create preconfigured connectors
-
-If you are running {kib} on-prem, you can define connectors by
-adding `xpack.actions.preconfigured` settings to your `kibana.yml` file.
-For example:
-
-[source,text]
+Method::    The HTTP request method, either `post`(default) or `put`.
+URL::       The request URL. If you are using the <<action-settings,`xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
+Authentication::
+The authentication type: none, basic, or SSL.
+If you choose basic authentication, you must provide a user name and password.
+If you choose SSL authentication, you must provide SSL server certificate authentication data in a CRT and key file format or a PFX file format. You can also optionally provide a passphrase if the files are password-protected.
+HTTP headers::   A set of key-value pairs sent as headers with the request.
+Certificate authority::
+CA file:::
+The certificate authority (CA) file that is used to sign and validate certificates.
```

Comment:
Not sure it's entirely clear that the CA file doesn't necessarily apply to SSL authentication, it also applies when configuring an HTTPS webhook that uses a self-signed certificate on the server side. Below references to the "provided certificate" may contribute to this confusion.

Reply:
For example, an HTTPS webhook that uses None or Basic authentication may also have its own server-side certificate signed by the user's own CA instead of a public CA. In this case the user will need to provide that CA file, or Kibana will throw a self-signed certificate error.

Reply:
Thanks, I've updated the CA descriptions in the docs and in the OAS files. If it's still not accurate, just let me know!